cfn-resources/serverless-instance/docs/serverlessinstanceprivateendpointendpoint.md

```diff
@@ -0,0 +1,59 @@
+# MongoDB::Atlas::ServerlessInstance ServerlessInstancePrivateEndpointEndpoint
```

Comment:
Typo with PrivateEndpointEndpoint?

Reply:
This not typo, we have a endpoint property inside the endpoint Object